sdk/storage/storage-blob-changefeed/rollup.config.js

```diff
@@ -1,17 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-import * as base from "./rollup.base.config";
+import { makeConfig } from "@azure/dev-tool/shared-config/rollup";
 
-const inputs = [];
-
-if (!process.env.ONLY_BROWSER) {
-  inputs.push(base.nodeConfig());
-}
-
-// Disable this until we are ready to run rollup for the browser.
-// if (!process.env.ONLY_NODE) {
-//   inputs.push(base.browserConfig());
-// }
-
-export default inputs;
+export default makeConfig(require("./package.json"), {
+  // Disable this until we are ready to run rollup for the browser.
```

Comment:
Question, I know it's pre-existing, but Is there an issue tracking this work?

Reply:
yeah https://github.com/Azure/azure-sdk-for-js/issues/11036